node.js

```diff
@@ -1,3 +1,11 @@
+/**
+ * Implements a node in a Chord, per Stoica et al., ca 2001. 
+ * 
+ * @requires gRPC
```

Comment:
What does this annotation communicate? We can see that grpc is required in the import a few lines down.